lxd/db/node.go

```diff
@@ -1047,6 +1047,70 @@ func (c *ClusterTx) GetNodeOfflineThreshold(ctx context.Context) (time.Duration,
 	return threshold, nil
 }
 
+func paramList(count int) string {
```

Comment:
Can we use `lxd/db/query.Params()` here?

Also if these are generic query helper functions I think it would be useful to move them into `lxd/db/query/expr.go`.